src/components/modals/EntityCreatorEditor/EntityCreatorContainer.tsx

```diff
@@ -351,6 +345,8 @@ class Container extends React.Component<Props, ComponentState> {
     onGetNameErrorMessage = (value: string): string => {
         const { intl } = this.props
 
+        const isNameUnrestricted = this.state.entityTypeVal !== CLM.EntityType.LUIS && this.state.entityTypeVal !== CLM.EntityType.LOCAL && this.state.entityTypeVal !== CLM.EntityType.ENUM
```

Comment:
I'd rather name this "isPrebuilt" as that explains exactly what is being calculated

Reply:
Threading the middle ground needle between reviewer comments